[2022-04-03 22:27:10,551] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: python_operator_univ_flores.dataframe_task manual__2022-04-04T03:26:48.543456+00:00 [queued]>
[2022-04-03 22:27:10,559] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: python_operator_univ_flores.dataframe_task manual__2022-04-04T03:26:48.543456+00:00 [queued]>
[2022-04-03 22:27:10,559] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-04-03 22:27:10,559] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-04-03 22:27:10,559] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-04-03 22:27:10,730] {taskinstance.py:1264} INFO - Executing <Task(_PythonDecoratedOperator): dataframe_task> on 2022-04-04 03:26:48.543456+00:00
[2022-04-03 22:27:10,734] {standard_task_runner.py:52} INFO - Started process 2948 to run task
[2022-04-03 22:27:10,739] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'python_operator_univ_flores', 'dataframe_task', 'manual__2022-04-04T03:26:48.543456+00:00', '--job-id', '325', '--raw', '--subdir', 'DAGS_FOLDER/ET_Univ_De_Flores_dag.py', '--cfg-path', '/tmp/tmpjhxedh2p', '--error-file', '/tmp/tmp372cxxwd']
[2022-04-03 22:27:10,740] {standard_task_runner.py:77} INFO - Job 325: Subtask dataframe_task
[2022-04-03 22:27:10,939] {logging_mixin.py:109} INFO - Running <TaskInstance: python_operator_univ_flores.dataframe_task manual__2022-04-04T03:26:48.543456+00:00 [running]> on host LAPTOP-5HJG55H7.localdomain
[2022-04-03 22:27:11,259] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=python_operator_univ_flores
AIRFLOW_CTX_TASK_ID=dataframe_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-04T03:26:48.543456+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-04T03:26:48.543456+00:00
[2022-04-03 22:27:11,261] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/decorators/base.py", line 134, in execute
    return_value = super().execute(context)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/lowenhard/airflow/dags/ET_Univ_De_Flores_dag.py", line 90, in dataframe_task
    process_dataframe()
  File "/home/lowenhard/airflow/dags/ET_Univ_De_Flores_dag.py", line 56, in process_dataframe
    df = pd.read_csv(os.path.join(folder, 'files/ET_Univ_.csv'))
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 680, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 575, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 933, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1217, in _make_engine
    self.handles = get_handle(  # type: ignore[call-overload]
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/io/common.py", line 789, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/home/lowenhard/airflow/files/ET_Univ_.csv'
[2022-04-03 22:27:11,312] {taskinstance.py:1272} INFO - Marking task as FAILED. dag_id=python_operator_univ_flores, task_id=dataframe_task, execution_date=20220404T032648, start_date=20220404T032710, end_date=20220404T032711
[2022-04-03 22:27:11,473] {standard_task_runner.py:89} ERROR - Failed to execute job 325 for task dataframe_task
Traceback (most recent call last):
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 180, in _run_raw_task
    ti._run_raw_task(
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/decorators/base.py", line 134, in execute
    return_value = super().execute(context)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/lowenhard/airflow/dags/ET_Univ_De_Flores_dag.py", line 90, in dataframe_task
    process_dataframe()
  File "/home/lowenhard/airflow/dags/ET_Univ_De_Flores_dag.py", line 56, in process_dataframe
    df = pd.read_csv(os.path.join(folder, 'files/ET_Univ_.csv'))
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 680, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 575, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 933, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1217, in _make_engine
    self.handles = get_handle(  # type: ignore[call-overload]
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/pandas/io/common.py", line 789, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/home/lowenhard/airflow/files/ET_Univ_.csv'
[2022-04-03 22:27:11,516] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-04-03 22:27:11,762] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
